UKHomeOffice/cop-ui
# Feature/cop 9749/ManageMyProfile-view

Adding the ability to view your profile through a react form, This pr is only for viewing profile details, Update functionality will be included in a later ticket.

Further detail outlined in the ticket.
https://support.cop.homeoffice.gov.uk/browse/COP-9749

**To Test** 
Check that you are able to view all fields in 'view profile', the "Are You A Civil Servant?" question will not display an answer as this was not previously a question in the old manage profile form, will be added along with the update ticket
